website/src/documentation/how-tos.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: section
+title: "How Tos"
```

Comment:
recommend "How-to resources"  and put the file in existing resources directory alongside the existing learning resources that has presentations, etc.

Reply:
- Moving both into a new "resources" directory
- Renaming this to "Learning Resources"
- Renaming the existing page to "Videos and Podcasts"
- Linking from "Learning Resources" to "Videos and Podcasts"